### PR TITLE
Add Scopes for Compute Engine & DNS

### PR DESCRIPTION
**Pull request by stephenplusplus**

### CODE REVIEW

File: authorization/readme.md [hunk 1]
```diff
@@ -8,9 +8,14 @@ If you are running this client on Google Compute Engine, we handle authorization
...
   * **BigQuery**
     * `https://www.googleapis.com/auth/bigquery`
+  * **Compute Engine**
+    * `https://www.googleapis.com/auth/compute`
   * **Datastore**
     * `https://www.googleapis.com/auth/datastore`
     * `https://www.googleapis.com/auth/userinfo.email`
+  * **DNS**
+    * `https://www.googleapis.com/auth/ndev.clouddns.readwrite`
+    * `https://www.googleapis.com/auth/cloud-platform`
```
Comment (dhermes) [inline authorization/readme.md]: Put this somewhere more generic.

Maybe comment on all the services covered by this scope?

Comment (stephenplusplus) [inline authorization/readme.md]: I'm actually getting this list from the scopes we use in node. All of these are exact copies of the scopes we give; no more or less. I wonder if we don't need `cloud-platform` for DNS, and only the first one. And maybe we never need `cloud-platform`? I don't remember where I got that from in the first place :confused:

Comment (dhermes) [inline authorization/readme.md]: It's listed in the discovery document for the service probably, but it's a very broad scope (like all of the cloud ones).

Comment (stephenplusplus) [inline authorization/readme.md]: Gotcha, thanks for pointing that out. Lmkwyt of the updates I just pushed.

**Comment by dhermes** [comment]
LGTM

**Comment by stephenplusplus** [comment]
Thanks for the review!

**Pull request merged**